frontend/components/DrawerLogo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+interface IconInterface {
+  navigationColor: string;
+  width: string;
+  height: string;
```

Comment:
We're basically reinventing Chakra styling props here.. Why not using the Chakra approach for custom icons? Read more [here](https://chakra-ui.com/docs/media-and-icons/icon#using-the-icon-component).
If you follow the chakra approach you'll get access to theme tokens and everything, without having to resort to custom logic